src/PKSim.Presentation/Presenters/Parameters/ParameterGroupsPresenter.cs

```diff
@@ -117,14 +117,14 @@ public void ActivateNode(ITreeNode node)
 
          _view.GroupCaption = node.FullPath(PKSimConstants.UI.DisplayPathSeparator);
 
-         bool alreadyLoaded = _parameterPresenterCache.Contains(node);
+         var alreadyLoaded = _parameterPresenterCache.Contains(node);
          _activePresenter = presenterFor(node);
          _settings.SelectedNodeId = node.Id;
 
          //this needs to be done before editing the parameters
          _view.ActivateView(_activePresenter.BaseView);
 
-         if (alreadyLoaded) return;
+         if (alreadyLoaded && !_activePresenter.AlwaysRefresh) return;
```

Comment:
That's where the flag is used. So far only the UserDefinedParameterPresenters is the one returning true. This ensure that the Edit is called everytime the node is being clicked